### PR TITLE
Add wmcb e2e tests to wmco

### PR DESCRIPTION
**Pull request by aravindhp**
Add CI plumbing that will result in Windows Machine Config Bootstrapper tests being run. The Makefile target was added in https://github.com/openshift/windows-machine-config-operator/pull/61.

- Manually added jobs based on the existing test-e2e-tools job

### CODE REVIEW

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml [hunk 1]
```diff
@@ -33,3 +33,7 @@ tests:
   commands: make verify-all
   container:
     from: src
+- as: e2e-wmcb
```
Comment (ravisantoshgudimetla) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml]: Perhaps something to signify `ci`?

File: ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml [hunk 2]
```diff
@@ -48,6 +48,75 @@ presubmits:
...
+        - --org=openshift
+        - --repo=windows-machine-config-operator
+        - --resolver-address=http://ci-operator-configresolver
+        - --secret-dir=/usr/local/e2e-wmcb-cluster-profile
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=e2e-wmcb
+        - --template=/usr/local/e2e-wmcb
+        command:
+        - ci-operator
+        env:
```
Comment (suhanime) [inline ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml]: We might want to add Cluster_Network_Type as [OVNKubernetes](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master-presubmits.yaml#L27).

Comment (ravisantoshgudimetla) [reply ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml]: +1 to what @suhanime found.

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml (hunk 1 above)
Comment (aravindhp) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml]: Wouldn't that be stating the obvious? I prefer seeing `ci/prow/e2e-wmcb`. This is a good pattern and is being followed in other repos.

File: ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml (hunk 2 above)
Comment (aravindhp) [reply ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml]: I will remove the config and add the job manually to make this happen. I am assuming there is no way to feed the `ci-operator` generator with options to make this happen automatically.

**Comment by aravindhp** [comment]
@ravisantoshgudimetla @suhanime I have addressed your comments. PTAL.

File: ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml [hunk 3]
```diff
@@ -48,6 +48,76 @@ presubmits:
...
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/e2e-wmcb
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
```
Comment (ravisantoshgudimetla) [inline ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml]: why is this needed?
